src/mainwindow/mainwindow.h

```diff
@@ -43,11 +44,12 @@ class MainWindow : public QMainWindow
     ProjectViewerDock *mpProjectViewerDock;
     ChatWindowDock *mpChatWindowDock;
     BottomPanelDock *mpBottomPanelDock;
-    QScopedPointer<DocumentManager> mpDocumentManager;    
     Browser *mDocumentationBrowser;
     QScopedPointer<PaletteConfigurator> mpPaletteConfigurator;
     Connection *db;
+    DocumentManager *mpDocumentManager;
     FileDb* dbFileManager;
+    CompilerControler * compilerControler;
```

Comment:
Code both naming and pointer sign conventions

Reply:
ok